app/controllers/ems_common.rb

```diff
@@ -464,6 +464,10 @@ def button
       javascript_redirect :controller         => "cloud_volume",
                           :action             => "new",
                           :storage_manager_id => params[:id]
+    elsif params[:pressed] == "cloud_volume_snapshot_create"
+      javascript_redirect :controller => "cloud_volume",
+                          :action     => "snapshot_new",
+                          :id         => find_checked_ids_with_rbac(CloudVolume).first
```

Comment:
@himdel since we are at it, should we change all other to the proposed function? We honestly were using other `if`s in `ems_common` as an example.